app/views/campaign_pages/index.slim

```diff
@@ -11,7 +10,7 @@
           th Actions
           th
       tbody
-        - @campaign_pages.each do |page|
+        - [*@campaign_pages].each do |page|
```

Comment:
It throws an error if search filters out all entries from the collection (if no campaign page matches the search criteria). This allows to render the page in that scenario (there's just nothing to iterate over). I'm sure there are other ways to take care of this, though.